commit b0c3f5df92913de6b9936cb0707cf1c8cdad66a2
Author: Sean Christopherson <seanjc@google.com>
Date:   Fri Jan 10 16:50:44 2025 -0800

    KVM: selftests: Macrofy vm_get_stat() to auto-generate stat name string
    
    Turn vm_get_stat() into a macro that generates a string for the stat name,
    as opposed to taking a string.  This will allow hardening stat usage in
    the future to generate errors on unknown stats at compile time.
    
    No functional change intended.
    
    Link: https://lore.kernel.org/r/20250111005049.1247555-5-seanjc@google.com
    Signed-off-by: Sean Christopherson <seanjc@google.com>

diff --git a/tools/testing/selftests/kvm/x86/dirty_log_page_splitting_test.c b/tools/testing/selftests/kvm/x86/dirty_log_page_splitting_test.c
index 2929c067c207..b0d2b04a7ff2 100644
--- a/tools/testing/selftests/kvm/x86/dirty_log_page_splitting_test.c
+++ b/tools/testing/selftests/kvm/x86/dirty_log_page_splitting_test.c
@@ -41,9 +41,9 @@ struct kvm_page_stats {
 
 static void get_page_stats(struct kvm_vm *vm, struct kvm_page_stats *stats, const char *stage)
 {
-	stats->pages_4k = vm_get_stat(vm, "pages_4k");
-	stats->pages_2m = vm_get_stat(vm, "pages_2m");
-	stats->pages_1g = vm_get_stat(vm, "pages_1g");
+	stats->pages_4k = vm_get_stat(vm, pages_4k);
+	stats->pages_2m = vm_get_stat(vm, pages_2m);
+	stats->pages_1g = vm_get_stat(vm, pages_1g);
 	stats->hugepages = stats->pages_2m + stats->pages_1g;
 
 	pr_debug("\nPage stats after %s: 4K: %ld 2M: %ld 1G: %ld huge: %ld\n",